css/helpers/square/_square.scss

```diff
@@ -0,0 +1,4 @@
+@mixin square($w,$h){
+    width: $w;
+    height: $h;
```

Comment:
po pierwsze nie uzywajmy takich skrotow :P, oszczedziles 2sekundy zycia i kilka liter, a sam kod jest przez to duzo mniej czytelny

po drugie zerknij na nagranie z zajec, pokazywalismy tam ze dla $height mozemy ustawic wartosc domyslna, wtedy jak bedziesz chcial stworzyc "kwadrat" to bedzie mogl podac w mixine tylko jedna wartosc i zostanie ona przypisana i do width i do height